include/ur_client_library/ur/ur_driver.h

```diff
@@ -265,13 +289,28 @@ class UrDriver
    */
   void setKeepaliveCount(const uint32_t& count);
 
+  /*!
+   * \brief Register a callback for the robot-based trajectory execution completion.
+   *
+   * One modes of robot control is to forward a complete trajectory to the robot for execution.
+   * When the execution is done, the callback function registered here, will be triggered.
+   *
+   * \param trajectory_done_cb Callback function that will be triggered in the event of finishing
+   * a trajectory execution.
```

Comment:
```suggestion
   * a trajectory execution
```
Consistency with other param documentations